thirdeye/thirdeye-frontend/app/helpers/set-has-not.js

```diff
@@ -0,0 +1,13 @@
+import Ember from 'ember';
+
+/**
+ * Returns true if the set does not contain the value
+ * @param {Object} set - the set
+ * @param {String} value - the value to check for
+ * @return {Boolean} !set.has(value)
+ */
+export function setHasNot([set = new Set(), value = null]) {
+  return !set.has(value);
+}
+
```

Comment:
I think lodash has `_reject`, which does what you're writing: https://lodash.com/docs/4.17.4#reject